src/wagtail_bynder/management/commands/base.py

```diff
@@ -17,11 +17,53 @@ class BaseBynderSyncCommand(BaseCommand):
     page_size: int = 200
     model: ModelBase = None
 
-    # Limits how far in the past to look for modified assets
-    # TODO: Make this a command-line option
-    modified_within_days: int = 1
+    def add_arguments(self, parser):
+        parser.add_argument(
+            "--minutes",
+            type=int,
+            help=_(
+                "The number of minutes into the past to look for asset "
+                "modifications."
+            ),
+        )
+        parser.add_argument(
+            "--hours",
+            type=int,
+            help=_(
+                "The number of hours into the past to look for asset "
+                "modifications (takes prescedence over 'minutes')"
+            ),
+        )
+        parser.add_argument(
+            "--days",
+            type=int,
+            help=_(
+                "The number of days in the past to look for asset "
+                "modifications (takes prescedence over 'hours' and 'minutes')"
+            ),
+        )
 
     def handle(self, *args, **options):
+        # Default timespan to 1 day (1440 minutes)
+        minutes = options.get("minutes")
+        hours = options.get("hours")
+        days = options.get("days") or 1
+        if hours:
+            timespan = timezone.timedelta(hours=hours)
+            timespan_desc = f"{hours} hour(s)"
+        elif minutes:
+            timespan = timezone.timedelta(minutes=minutes)
+            timespan_desc = f"{minutes} minute(s)"
+        else:
```

Comment:
the `--days` help text says this takes precendence over hours and minutes, but this doesn't quite hold true.

This should be 


```python
minutes = options.get("minutes")
hours = options.get("hours")
days = options.get("days")

if days:
    timespan = timezone.timedelta(days=days)
    timespan_desc = f"{days} day(s)"
elif hours:
    timespan = timezone.timedelta(hours=hours)
    timespan_desc = f"{hours} hour(s)"
elif minutes:
    timespan = timezone.timedelta(minutes=minutes)
    timespan_desc = f"{minutes} minute(s)"
else:
    timespan = timezone.timedelta(days=1)
    timespan_desc = "1 day(s)"
```

note: while the last else should use "1 day" I left it to "1 day(s)" for consistency with the others